src/core/handle.ml

```diff
@@ -276,9 +276,15 @@ let handle_cmd : sig_state -> p_command -> sig_state * proof_data option =
         | P_config_unif_rule(h)   ->
             (* Approximately same processing as rules without SR checking. *)
             let pur = (scope_rule ss h).elt in
+            let rhs =
+              Bindlib.unbox (Bindlib.bind_mvar pur.pr_vars pur.pr_rhs)
+            in
+            let (xs, rhs) = Bindlib.unmbind rhs in
```

Comment:
bind followed by unbind could be avoided by just unboxing pur.pr_rhs (binders can be unboxed too).